packages/engine/Source/Widget/CesiumWidget.js

```diff
@@ -387,6 +430,72 @@ function CesiumWidget(container, options) {
       }
     };
     scene.renderError.addEventListener(this._onRenderError);
+
+    let dataSourceCollection = options.dataSources;
+    let destroyDataSourceCollection = false;
+    if (!defined(dataSourceCollection)) {
+      dataSourceCollection = new DataSourceCollection();
+      destroyDataSourceCollection = true;
+    }
+
+    const dataSourceDisplay = new DataSourceDisplay({
+      scene: scene,
+      dataSourceCollection: dataSourceCollection,
+    });
+
+    const eventHelper = new EventHelper();
+    this._dataSourceChangedListeners = {};
+    this._automaticallyTrackDataSourceClocks = defaultValue(
+      options.automaticallyTrackDataSourceClocks,
+      true
+    );
```

Comment:
We are phasing out `defaultValue`, see https://github.com/CesiumGS/cesium/pull/12207. I know this was just copied over but to avoid future merge conflicts can you replace usages like this in "new" code with `??`?
```suggestion
    this._automaticallyTrackDataSourceClocks = 
      options.automaticallyTrackDataSourceClocks ?? true;
```

Reply:
Sure!